stable/admin/Chart.yaml

```diff
@@ -12,5 +12,5 @@ maintainers:
   email: support@nuodb.com
 engine: gotpl
 icon: https://raw.githubusercontent.com/nuodb/nuodb-helm-charts/master/images/nuodb.svg
-appVersion: 5.0.3
+appVersion: 5.0.4
```

Comment:
I guess that we need to bump the pinned `tag` in the values.yaml so that to is accurate.

Reply:
Good point.